erddap/templates/private/dataset.deployment.xml

```diff
@@ -301,184 +301,35 @@
-            <att name="long_name">precise_lon Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>profile_time_qc</sourceName>
-        <destinationName>time_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">profile_time Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>profile_lat_qc</sourceName>
-        <destinationName>latitude_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">latitude Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>profile_lon_qc</sourceName>
-        <destinationName>longitude_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">longitude Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>time_uv_qc</sourceName>
-        <destinationName>time_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">time_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>lat_uv_qc</sourceName>
-        <destinationName>lat_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">lat_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>lon_uv_qc</sourceName>
-        <destinationName>lon_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">lon_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>u_qc</sourceName>
-        <destinationName>u_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">u Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>v_qc</sourceName>
-        <destinationName>v_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">v Variable Quality Flag</att>
+            <att name="long_name">{{dest_var_remaps.get(req_var, req_var)}} Variable Quality Flag</att>
```

Comment:
Does the output from ```{{dest_var_remaps.get(req_var, req_var)}}``` result in the variable name + '_qc'? If so I think we need to drop the ```_qc``` in the long name.

E.G.
```
<destinationName>u_qc</destinationName>
<att name="long_name">u Variable Quality Flag</att>

```